### PR TITLE
Resize uploaded images to fixed dimension

### PR DESCRIPTION
**Pull request by Devorein**
### WHY

[Card](https://app.charmverse.io/charmverse/resize-all-uploaded-images-7723593554479748)

### CODE REVIEW

File: lib/file/constants.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
 export const DEFAULT_MAX_FILE_SIZE_MB = 20;
 export const FORM_DATA_FILE_PART_NAME = 'uploadedFile';
 export const DEFAULT_IMAGE_SIZE = 512;
+export const FORM_DATA_IMAGE_RESIZE_TYPE = 'resizeType';
+export type ResizeType = 'emoji' | 'artwork';
```
Comment (mattcasey) [inline lib/file/constants.ts:5]: suggestion: use enum for this

**Review by mattcasey — APPROVED** [review]

File: pages/api/image/resize.ts [hunk 2]
```diff
@@ -6,14 +6,25 @@ import nc from 'next-connect';
...
+  FORM_DATA_IMAGE_RESIZE_TYPE
+} from 'lib/file/constants';
 import { onError, onNoMatch, requireUser } from 'lib/middleware';
 import { withSessionRoute } from 'lib/session/withSession';
 
 const handler = nc<NextApiRequest, NextApiResponse>({ onError, onNoMatch });
 
 handler.use(requireUser).post(resizeImage);
 
+const ResizedTypeWidth: Record<ResizeType, number> = {
```
Comment (mattcasey) [inline pages/api/image/resize.ts:23]: should either be resizedTypeWidth or RESIZED_TYPE_WIDTH

File: lib/file/constants.ts (hunk 1 above)
Comment (Devorein) [reply lib/file/constants.ts:5]: Yeah sure. Updated.

File: pages/api/image/resize.ts (hunk 2 above)
Comment (Devorein) [reply pages/api/image/resize.ts:23]: Good point. Updated.

**Pull request merged**